clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/CreateServerGroupAtomicOperationSpec.groovy

```diff
@@ -1853,6 +1853,39 @@ class CreateServerGroupAtomicOperationSpec extends CommonAtomicOperation {
     result.getExecutionRoleArn() == "arn:aws-us-gov:iam:123123123123:role/test-role"
   }
 
+  def 'should return valid task def from spelProcessedArtifact'() {
+    given:
+    def credentials = Mock(NetflixAssumeRoleAmazonCredentials) {
+      getName() >> { "test" }
+      getRegions() >> { [new AmazonCredentials.AWSRegion('us-west-1', ['us-west-1a', 'us-west-1b'])] }
+      getAssumeRole() >> { 'arn:aws-us-gov:iam:123123123123:role/test-role' }
+      getAccountId() >> { 'test' }
+    }
+    def resolvedArtifact = createResolvedArtifact()
+    def containerDef = createContainerDef()
+    def registerTaskDefRequest = createRegisterTaskDefRequest(containerDef)
+    def description = createDescription(resolvedArtifact);
+    description.isEvaluateTaskDefinitionArtifactExpressions() >> true
+    Map<Object, Object> map = new HashMap<>()
```

Comment:
I know it's small/simple, but can we name this something more descriptive ("spelArtifactMap", etc.)

Reply:
also: maybe we can put a different execution role in here and then assert on it below to make sure the valued from this object is what's being used.